mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -50,7 +50,7 @@ interface RenderModalContentParams {
  * TODO: use modal instead of confirm task #305
  */
 export const renderModalContent = (params: RenderModalContentParams) => {
-  const isUserWantToDeleteJobDone = confirm(`Are you sure that you want to delete jobDone "${params.description}"?`);
+  const isUserWantToDeleteJobDone = confirm(params.description);
   !!isUserWantToDeleteJobDone && params.onOk();
```

Comment:
Remove !!, confirm returns boolean

Reply:
Also rename const

Reply:
Removed !! and renamed const to isOkClicked